docs/_posts/murat-gunay/2021-12-25-xlm_roberta_large_token_classifier_conll03_de.md

```diff
@@ -0,0 +1,125 @@
+task: Named Entity Recognition
+language: de
+edition: Spark NLP 3.3.4
+spark_version: 2.4
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model was imported from `Hugging Face` and it's been fine-tuned for the German language, leveraging `XLM-RoBERTa` embeddings and `XlmRobertaForTokenClassification` for NER purposes.
+
+## Predicted Entities
+
+`PER`, `ORG`, `LOC`, `MISC`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/public/models/xlm_roberta_large_token_classifier_conll03_de_3.3.4_2.4_1640443988652.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+documentAssembler = DocumentAssembler()\
+      .setInputCol("text")\
+      .setOutputCol("document")
+
+sentenceDetector = SentenceDetectorDLModel.pretrained("sentence_detector_dl", "xx")\
+       .setInputCols(["document"])\
+       .setOutputCol("sentence")
+
+tokenizer = Tokenizer()\
+      .setInputCols(["sentence"])\
+      .setOutputCol("token")
+
+tokenClassifier = XlmRoBertaForTokenClassification.pretrained("xlm_roberta_large_token_classifier_conll03", "de"))\
+  .setInputCols(["sentence",'token'])\
+  .setOutputCol("ner")
+
+ner_converter = NerConverter()\
+      .setInputCols(["sentence", "token", "ner"])\
+      .setOutputCol("ner_chunk")
+      
+nlpPipeline = Pipeline(stages=[documentAssembler, sentenceDetector, tokenizer, tokenClassifier, ner_converter])
+
+empty_data = spark.createDataFrame([[""]]).toDF("text")
+
+model = nlpPipeline.fit(empty_data)
+text = """Ich bin Michael, ich lebe in Stuttgart und habe im Mercedes-Benz Werk gearbeitet."""
+result = model.transform(spark.createDataFrame([[text]]).toDF("text"))
+```
+```scala
+val documentAssembler = DocumentAssembler()
+      .setInputCol("text")
+      .setOutputCol("document")
+
+val sentenceDetector = SentenceDetectorDLModel.pretrained("sentence_detector_dl", "xx")
+       .setInputCols(Array("document"))
+       .setOutputCol("sentence")
+
+val tokenizer = Tokenizer()
+      .setInputCols(Array("sentence"))
+      .setOutputCol("token")
+
+val tokenClassifier = XlmRoBertaForTokenClassification.pretrained("xlm_roberta_large_token_classifier_conll03", "de"))\
+  .setInputCols(Array("sentence","token"))\
+  .setOutputCol("ner")
+
+ner_converter = NerConverter()\
+      .setInputCols(Array("sentence", "token", "ner"))\
+      .setOutputCol("ner_chunk")
+      
+val pipeline = new Pipeline().setStages(Array(documentAssembler, sentenceDetector, tokenizer, tokenClassifier, ner_converter))
+
+val example = Seq.empty["Ich bin Michael, ich lebe in Stuttgart und habe im Mercedes-Benz Werk gearbeitet."].toDS.toDF("text")
```

Comment:
Can we pick a real German sentence from Wikipedia?

Reply:
@josejuanmartinez 
I changed the German text; it took more than 30 mins to find a good sentence from Wikipedia. everything on the page are German, you know... a little bit time consuming. 😄

Reply:
@murat-gunay  we always need to do it. I showed the previous sentence to a German speaker and he was horrified :)